include/mbedtls/ssl.h

```diff
@@ -191,6 +191,7 @@
 
 /* Elliptic Curve Groups (ECDHE) */
 #define MBEDTLS_SSL_IANA_TLS_GROUP_NONE               0
+#define MBEDTLS_SSL_IANA_TLS_GROUP_SECT163K1     0x0001 /* RFC 4492 */
```

Comment:
Will this group be deprecated? Or do tls1.2 and tls1.3 protocol declare to support it?

Reply:
It is IANA value from rfc4492. I did not found deprecated description from RFC.
Here is used for detect EC group. Just put IANA value.